code/datums/tips/tips.dm

```diff
@@ -0,0 +1,341 @@
+		"As a Hangar Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, phoron sheets, rare seeds from hydroponics, and more!",
+		"As the Operations Manager, be sure to check the manifests on crates you receive to make sure all the info is correct. If there's a mistake, stamp the manifest DENIED and send it back in a crate with the items untouched for a refund!",
+		"As a Shaft Miner, always have a GPS on you, so a fellow miner or cyborg can come to save you if you die."
+	)
+
+
+/*
+	SERVICE TIPS
+*/
+
+/datum/tip/service
+	category = CATEGORY_SERVICE
+	messages = list(
+		"As the Chaplain, your bible is also a container that can store small items.",
+		"As the Chaplain, your null rod has a lot of functions: it can convert water into holy water, which has a few practical applications. The rod itself can be used against evil and supernatural elements aswell.",
+		"As the Chaplain, you are much more likely to get a response by praying to the gods than most people. To boost your chances, make altars with colorful crayon runes, lit candles, and wire art.",
+		"As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic seeds. These seeds can alternatively be ordered from cargo.",
+		"As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.",
+		"As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.",
+		"As a Cook, any food you make will be much healthier than the junk food found in vendors. The crew will thank you for eating healthier, eventually.",
+		"As the Bartender, the drinks you start with only give you the basics. If you want more advanced mixtures, look into working with chemistry, hydroponics, or even mining for things to grind up and throw in!",
+		"As the Bartender, you can use a circular saw on your shotgun to make it easier to store.",
+		"As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer or order another from Cargo.",
+		"As a Janitor, mousetraps can be used to create bombs or booby-trap containers.",
+		"As a Janitor, you have an excuse to be almost anywhere.",
+		"As the Librarian, be sure to keep the shelves stocked and the library clean for the crew."
+	)
+
+/*
+	ANTAG TIPS
+*/
+
+/datum/tip/antag
+	category = CATEGORY_ANTAG
+	messages = list(
+		"As a Traitor, the cryptographic sequencer (emag) can not only open doors, but also lockers, crates, APCs and more. It can hack cyborgs, and even cause bots to go berserk. Use it on the right machines, and you can even order more traitor gear or contact the Syndicate. Experiment!",
```

Comment:
```suggestion
		"As a Traitor, the cryptographic sequencer (emag) can not only open doors, but also lockers, crates, APCs and more. It can hack cyborgs, and even cause bots to go berserk. Use it on the right machines, and you can even order more traitor gear or contact an anti-corporate group willing to help you. Experiment!",
```